src/NATS.Client.KeyValueStore/Internal/NatsKVWatcher.cs

```diff
@@ -193,6 +193,11 @@ private async Task CommandLoop()
                                         _ => operation,
                                     };
                                 }
+
+                                if (headers is { Code: 100, MessageText: "FlowControl Request" })
+                                {
+                                    await msg.ReplyAsync(cancellationToken: _cancellationToken);
```

Comment:
need `continue;` afterwards?  Or is it correct to fall-through and continue processing?

Reply:
nothing happens after that point but makes sense to continue